ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Is this used? If so (even if just by you) this is fine. Otherwise I'd delete the whole directory.